contrib/verifybinaries/verify.sh

```diff
@@ -86,18 +86,23 @@ fi
 WGETOUT=$(wget -N "$HOST1$BASEDIR$SIGNATUREFILENAME" 2>&1)
 
 #and then see if wget completed successfully
+# shellcheck disable=SC2181
 if [ $? -ne 0 ]; then
```

Comment:
I believe we can just do:
```sh
if ! WGETOUT=$(wget -N "$HOST1$BASEDIR$SIGNATUREFILENAME" 2>&1); then
```

See: https://github.com/koalaman/shellcheck/issues/782

Reply:
Fixed.